test/images/agnhost/openidmetadata/openidmetadata.go

```diff
@@ -48,32 +49,63 @@ var CmdTestServiceAccountIssuerDiscovery = &cobra.Command{
 }
 
 var (
-	tokenPath          string
-	audience           string
-	inClusterDiscovery bool
+	tokenPath                    string
+	audience                     string
+	deprecatedInClusterDiscovery bool
```

Comment:
I'd suggest dropping this rather than keeping it registered and ignored

Reply:
Done